rst2pdf/extensions/preprocess_r2p.py

```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
-# An extension module for rst2pdf
+# SPDX-License-Identifier: MIT
 # Copyright 2010, Patrick Maupin
```

Comment:
What does it mean to have our license terms and a copyright declaration in the same file?

Reply:
@pmaupin: Are you willing to relicense your work in `preprocess_r2p.py` to the rst2pdf project as a whole under the MIT licence?